tests/test_dori.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+import os
+import numpy as np
+from pyscf.dft import numint
+from qstack import compound
+from qstack.fields.dm import make_grid_for_rho
+from qstack.fields.dori import dori, dori_on_grid, compute_rho
+
+
+def grad_num(func, grid, eps=1e-4, **kwargs):
```

Comment:
not sure why this function is outside of test_dori_deriv
and also why it loops over the grid points when the existing implementation of this in `dori.py` executes `func()` over the whole grid at once

Reply:
for readability and so that it would be easier to adapt it for some other tests in the future

Reply:
I've just learned that `numpy.gradient` exists... should we switch to it?

Reply:
we could, but the algorithm used here and in np.gradient are not the same I think.
yeah numpy uses 3 points (x, x-e, x+e) while you use four (x-2e,x-e,x+e,x+2e)

Reply:
right! 
 I'll keep the 5-point stencil then